From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_git@polytechnique.org>
Date: Mon, 30 May 2016 20:13:57 +0200
Subject: [PATCH] {NOT UPSTREAMABLE} lib/test_hash: make clang happy

clang does not like the initializer:

    lib/test_hash.c:146:40: error: suggest braces around initialization
    of subobject [-Werror,-Wmissing-braces]
        u32 string_or = 0, hash_or[2][33] = { 0 };
                                              ^
                                              {}
---
 lib/test_hash.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/test_hash.c b/lib/test_hash.c
index 66c5fc8351e8..7b2291cfbf09 100644
--- a/lib/test_hash.c
+++ b/lib/test_hash.c
@@ -143,7 +143,7 @@ static int __init
 test_hash_init(void)
 {
 	char buf[SIZE+1];
-	u32 string_or = 0, hash_or[2][33] = { 0 };
+	u32 string_or = 0, hash_or[2][33] = { { 0 } };
 	unsigned tests = 0;
 	unsigned long long h64 = 0;
 	int i, j;
-- 
